### DISCUSSION TITLE
keybindings comment部分出问题

### DISCUSSION THREAD

**Discussion by EvilChan** [nshen/learn-neovim-lua#167] [General]
-- 代码注释插件
pluginKeys.comment = {
  ...,
  opleader = {
    line = "gc",
    --bock = "gb",
    block = "gb", -- fix
  },
}
-- Ctrl + /
map("n", "<C-_>", "gcc", { noremap = false })
-- map("v", "<C-_>", "gcc", { noremap = false })
map("v", "<C-_>", "gc", { noremap = false }) -- fix

VISUAL 模式下使用注释，多出一个c出来，导致内容突然删除并进入编辑模式
大佬，使用的时候不慌的吗，还是没用上，手动狗头[doge][doge][doge]

**Comment by nshen** [comment]
可能我写错了？现在我是 normal 模式下 gcc / gbc ，visual 下 gc / gb

**Reply by EvilChan** [reply]
注释的都是错误的地方，无伤大雅

**Reply by nshen** [reply]
好的谢谢，我现在在用最新的配置在insisvim那个项目里，如果有错了可以对比一下那边
